client/app/bundles/buttons/ButtonFilled.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import css from './button.scss';
+
+class ButtonFilled extends React.Component {
+   render() {
+       
+       
+      return (
+          <div>
+         <button className = {css.buttonFilled}>
+            Learn More
```

Comment:
This should be in our i18n files and referenced that way! Let me know if you need help figuring that out on Slack :D